profiles/templates/components/skills-form.html

```diff
@@ -0,0 +1,36 @@
+{% load static %}
+
+<div id="skills">
+  <h1 class="block text-violet-700 text-2xl font-bold mb-2">Skills</h1>
+  <p class="block text-gray-700 text-sm">Please enter your professional skills</p>
+  <p class="block text-gray-700 text-sm mb-4">A minimum of 8 skills is recommended</p>
+  <label for="new-skill-text" class="block text-gray-700 text-sm font-bold mb-2">Skills</label>
+  <form method="post" id="skill-list-form">
+    {% csrf_token %}
+    <ul id="skill-list" class="w-[50%] pl-6 mb-3">
+      {% for skill in user.user_skills.all %}
+      <li id="skill-{{ skill.name }}" class="skill-item list-disc">
+        <span class="flex flex justify-between">
+          <span>{{ skill.display_name }}</span>
+          <button type="button" class="delete-skill" data-skill="{{ skill.name }}"><span
+              class="text-2xl">&times;</span></button>
+        </span>
+      </li>
+      {% endfor %}
+    </ul>
+  </form>
+  <!-- The inline form to add the skills -->
+  <form method="post" id="skill-form" class="w-[50%]">
+    {% csrf_token %}
+    <div class="flex">
+      <div class="relative w-full">
+        <input type="text" id="new-skill-text"
+          class="block p-2.5 w-full z-20 text-sm text-gray-900 bg-gray-50 rounded-l-lg rounded-e-lg rounded-s-gray-100 rounded-s-2 border border-gray-300 focus:ring-lime-300 focus:border-lime-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:border-lime-500"
```

Comment:
Suggestion (Non-blocking): I think it's best to remove any classes starting with `dark:`. We've decided we won't currently offer a dark mode, so it will be more consistent if they're removed.